src/controllers/login-controller.js

```diff
@@ -1,33 +1,58 @@
 const Accounts = require('../models/accounts');
-const {sendToTelegram} = require('../helpers/telegramBot');
-const  {tokenSing}= require('../helpers/generateToken');
-const {jsonErrors} = require('../helpers/exception-responses');
-const {ResponseREST} = require('../helpers/responses');
-const {Number} = require('../helpers/number');
-
+const {tokenSing}= require('../helpers/generateToken');
+const {StatusCodes} = require ('http-status-codes');
+const {responseServer, responseGeneral} = require('../helpers/response-result');
 
 const postLogin = async (request, response) => {
     const {username, password} = request.body;
-    const queryAccounts  = Accounts.where({ status: Number.ONE, username: username});
-    queryAccounts.findOne(function (error, accounts) {   
-        if (error){           
-            sendToTelegram(exception);
-            response.status(ResponseREST.SERVER_ERROR).json(jsonErrors(ResponseREST.SERVER_ERROR));
-        }     
-        if (accounts) {
-            const isValidPassword = await accounts.matchPassword(password);
-            if(isValidPassword){
-                const token = tokenSing(accounts);
-                return res.json({token: token});
+    await Accounts.findOne({username: username}, {_id:1, status:1, password:1, role:1, name:1, lastname:1, URLPhoto:1})
+    .then(async (account) =>{  
```

Comment:
Codacy has a fix for the issue: [Missing space after =>.](https://app.codacy.com/gh/Martha-Miroslava/APIBiziidiDiidxaza/pullRequest?prid=8267260)
```suggestion
    .then(async (account) => {  
```